pillar/components/sspl.sls

```diff
@@ -89,40 +48,6 @@ sspl:
     user: "xxxxx"
     password: "gAAAAABehkmVHStx337AN2g6OTALqA5BNejmWD6Nu__25DgYzauLz6iGLLCDzqs71pP7Z5AV-oEvnOPGKomJiTzyPwOHnmylEQ=="
     mgmt_interface: "cliapi"
-  REALSTORSENSORS:
-    polling_frequency: "30"
-  REALSTORPSUSENSOR:
-    threaded: "true"
-  REALSTORFANSENSOR:
-    threaded: "true"
-  REALSTORCONTROLLERSENSOR:
-    threaded: "true"
-  REALSTORDISKSENSOR:
-    threaded: "true"
-    polling_frequency_override: "0"
-  REALSTORSIDEPLANEEXPANDERSENSOR:
-    threaded: "true"
-  NODEHWSENSOR:
-    threaded: "true"
-    polling_interval: "30"
-  REALSTORLOGICALVOLUMESENSOR:
-    threaded: "true"
-  REALSTORENCLOSURESENSOR:
-    threaded: "true"
-  IEMSENSOR:
-    threaded: "true"
-    log_file_path: "/var/log/cortx/iem/iem_messages"
-    timestamp_file_path: "/var/cortx/sspl/data/iem/last_processed_msg_time"
-  SYSTEMDWATCHDOG:
-    threaded: "true"
-    smart_test_interval: "999999999"
-    run_smart_on_start: "False"
-  NODEHWACTUATOR:
-    ipmi_client: "ipmitool"
-  DATASTORE:
-    store_type: "consul"
-    consul_host: "127.0.0.1"
-    consul_port: "8500"
   RABBITMQINGRESSPROCESSOR:
```

Comment:
@shubhamseagate , Please take care of the comment given given in below discussion
https://github.com/Seagate/cortx-prvsnr/pull/591#discussion_r502813753